plugin/pkg/shoot/validator/admission.go

```diff
@@ -23,21 +24,29 @@ import (
 	"time"
 
 	"github.com/gardener/gardener/pkg/apis/core"
+	corev1alpha1 "github.com/gardener/gardener/pkg/apis/core/v1alpha1"
```

Comment:
```suggestion
	gardencorev1alpha1 "github.com/gardener/gardener/pkg/apis/core/v1alpha1"
```

We agreed to use `gardencorev1alpha1` to prevent naming conflicts and confusion with Kubernetes' `core/v1` API.